src/components/Accordion/AccordionItem.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Accordion.css';
+
+class AccordionItem extends React.Component {
+  constructor(props) {
+    super(props);
+    this.accordionItemToggle = this.accordionItemToggle.bind(this);
+    this.state = {
+      hidden: true,
+    };
+  }
+  accordionItemToggle() {
+    const currentState = this.state.hidden;
+    this.setState({ hidden: !currentState });
```

Comment:
Seems like this could be a single line arrow function without declaring the `currentState` const.

Reply:
How would that look?

Reply:
For example: `accordionItemToggle => this.setState({ hidden: !this.state.hidden });`